pom.xml

```diff
@@ -1360,7 +1360,26 @@
                                     <exclude>org.codehaus.plexus:plexus-utils</exclude>
                                     <exclude>com.google.guava:guava</exclude>
                                     <exclude>com.fasterxml.jackson.core:jackson-core</exclude>
+                                    <exclude>com.fasterxml.jackson.core:jackson-annotations</exclude>
```

Comment:
I'm not familiar with Maven and there are all sorts of upper bound dependency conflicts when adding `sn-pseudonymisation-lib`. Not sure if it's alright to just exclude them. Have you run into issues like this before? @Sylvain-Royer